training/training/core/dataset.py

```diff
@@ -12,10 +12,15 @@
 import torch
 from torch.utils.data import Dataset
 from torch.autograd import Variable
+from torchvision import datasets, transforms
+from torch.utils.data import DataLoader
+from enum import Enum
+import os
+import shutil
 
 
 class TrainTestDatasetCreator(ABC):
-    "Creator that creates train and test PyTorch datasets"
+    "Creator that creates train and test PyTorch datasets from a given dataset"
```

Comment:
is this proper convention for class docs?

Reply:
fixed